protos/dcs_mission.proto

```diff
@@ -29,6 +29,15 @@ service Mission {
 
   // DCT Function
   rpc JoinMission(MissionJoinRequest) returns (MissionJoinResponse) {}
+}
 
-  rpc StreamEvents(StreamEventsRequest) returns (stream Event) {}
+service Streamer {
+	// Streams DCS game generated Events.
+	// See https://wiki.hoggitworld.com/view/Category:Events
+	rpc Events(StreamEventsRequest) returns (stream Event) {}
```

Comment:
I'd actually tend towards keeping `Events` inside of the `Mission` service, and split the services by e.g. `Mission`, `Unit`, `Group`, ...

Reply:
That is fine, this was just an easy one to test with.

Reply:
Split more fully, as discussed in Discord we will make the services plural (Mission will stay singular).